package.json

```diff
@@ -50,7 +50,11 @@
     "prettier": "prettier --write src/index.js src/views/**/*.js src/models/**/*.js",
     "start": "webpack-dev-server --progress --open --config webpack.dev.js",
     "watch": "webpack --progress --config webpack.dev.js --watch",
-    "test": "jest"
```

Comment:
you need to update the `Makefile` so that `make test` points to one of the new commands so that the buildbot tests can continue to run.